src/components/Hamburger/Hamburger.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { hideVisually } from 'polished';
+
+const HAMBURGER_WIDTH = '16px';
```

Comment:
Why not using the property `iconSizes` of the theme?  We have this size in `kilo`

Reply:
Good point 👍